Mixpanel/MPNotificationViewController.m

```diff
@@ -163,7 +163,15 @@ - (void)setUpButtonView:(UIButton *)buttonView withData:(MPNotificationButton *)
 }
 
 - (void)buttonTapped:(UIButton *)button {
-    [self.delegate notificationController:self wasDismissedWithCtaUrl:((MPTakeoverNotification *)self.notification).buttons[button.tag].ctaUrl];
+    MPTakeoverNotification *takeoverNotification = (MPTakeoverNotification *)self.notification;
+    NSString *whichButton = @"primary";
+    if (takeoverNotification.buttons.count == 2) {
+        whichButton = button.tag == 0 ? @"secondary" : @"primary";
```

Comment:
just double checking that in android `buttonIndex=0` is `primary` and in iOS seems the other way around

Reply:
apparently it's right here and wrong on android